### DISCUSSION TITLE
Произошла непредвиденная ошибка при загрузке карты

### DISCUSSION THREAD

**Discussion by d-nikolaev** [cdek-it/wordpress#9] [Q&A] [answered]
Не работает Яндекс.Карта и сам плагин в целом.
Все ключи прописаны. По части яндекс карты получили вот такой ответ от яндекса.:

Библиотека [JS API](https://yandex.ru/dev/maps/jsapi/) и запросы к HTTP API [Геокодера](https://yandex.ru/dev/maps/geocoder) работают корректно, если при их подключении указан API-ключ.

Сейчас вы используете виджет СДЭК, который эти правила не учитывает, поэтому API-ключ в запросах не передаётся.

Пожалуйста, обратитесь к разработчику виджета с этой информацией. Они могут написать нам напрямую на [maps-api@support.yandex.ru](mailto:maps-api@support.yandex.ru), мы поможем разобраться.

**Comment by vermorag** [comment]
Добрый день. Вам наобходимо проверить, что 
- У вас установлена последняя версия плагина (на данный момент 3.20.1)
- Вы обновили ключи интеграции в плагине
- Вы сохранили обновленный ключ яндекса в плагине
- С момента обновления ключа яндекса прошло больше 1 часа

Если все вышеперечисленное было проделано, вы можете обратиться в чат технческой поддержки: https://t.me/cdek_it_bot
раздел Интеграция для интернет-магазинов. Понадобится диагностика вашего сайта.

**Comment by d-nikolaev** [comment]
1. Версия палагина: 3.20.1
2. Ключи: Дата обновления 10.06.2024 18:52:50
3. В плагине обновленный ключ.
4. С момента обновления ключа яндекс прошло более 24 часов.

**Reply by vermorag** [reply]
Тогда только в техническую поддержку, необходимо идентифицировать вас, как клиента и посмотреть что на сайте с плагином.

**Reply by d-nikolaev** [reply]
Можете дать email?

**Reply by vermorag** [reply]
В настоящий момент поддержка по почте не предоставляется по техническим причинам.

**Reply by vermorag** [reply]
Вижу, стоят константы 
`WP_MEMORY_LIMIT=40M`
И лимит памяти для скриптов стоит в 128M. 
Для вукоммерса минимум: [256M](https://woocommerce.com/document/server-requirements/)

Поэтому карту и не грузит.
Он отправляет несколько запросов на бэк для получения офисов. 1-2 стабильно падают (500 или 502), предполагаю, что как раз из-за памяти.

Сейчас в обход карты поставил ПВЗ отправки NSK65 (без адреса отправки или ПВЗ отправки расчета происходить не будет и на чекауте метод доставки не появится) и на чекауте все посчиталось.

**Reply by d-nikolaev** [reply]
Спасибо. Понял.

**Reply by d-nikolaev** [reply, marked as answer]
Добавил оперативной памяти, всё заработало.

**Comment by nxnwebdev** [comment]
Такая же проблема, настроили по инструкции, все ключи актуальные, версия плагина финальная, в админке не отображается ПВЗ на карте, сама карта при это прогружается, и нет никаких расчетов в чекауте.

**Reply by d-nikolaev** [reply]
Аналогичная история. Но у нас даже карта не грузится.

**Reply by vermorag** [reply]
Можете скриншот показать? Карта + блок выбора ПВЗ над ней.

**Reply by nxnwebdev** [reply]
![cdek](https://github.com/cdek-it/wordpress/assets/94235054/f95db0aa-6e35-4cbd-80a7-a0acc84ddff9)

**Reply by vermorag** [reply]
Сможете предоставить скриншот со ссылкой на данный issue + доступами в админ панель в телеграм бота поддержки?

**Reply by d-nikolaev** [reply]
Посмотрите у меня тоже. В чат выслал.

**Reply by vermorag** [reply]
@nxnwebdev Проверьте еще - раз у вас выбран только адрес отправки, значит и тарифы должны быть "от склада".
